mlir/test/Quantum/ConversionTest.mlir

```diff
@@ -179,7 +179,7 @@ func.func @custom_gate(%q0 : !quantum.bit, %p : f64) -> (!quantum.bit, !quantum.
     // CHECK: [[nullPtr:%.+]] = llvm.mlir.zero : !llvm.ptr
     // CHECK: [[true:%.+]] = llvm.mlir.constant(true) : i1
     // CHECK: [[zero:%.+]] = llvm.mlir.constant(0 : i64) : i64
-    // CHECK: [[struct:%.+]] = llvm.alloca %16 x !llvm.struct<(i1, i64, ptr, ptr)> : (i64) -> !llvm.ptr
+    // CHECK: [[struct:%.+]] = llvm.alloca %10 x !llvm.struct<(i1, i64, ptr, ptr)> : (i64) -> !llvm.ptr
```

Comment:
This should not be a fixed number since we don't know what `%10` is referring to. We should either:
- capture its creation if we care about where the value comes from
- capture an arbitrary value here if we don't care where it comes from (e.g. `{{%.+}}`)

Reply:
[af0377a](https://github.com/PennyLaneAI/catalyst/pull/585/commits/af0377ab3d9b21e8d9aeaaf2795a366bf1e4d96d)